test/common/EnvVars.hpp

```diff
@@ -28,6 +28,7 @@ namespace RcclUnitTesting
     int  maxRanksPerGpu; // Number of ranks using the same GPU     [UT_MAX_RANKS_PER_GPU]
     bool showTiming;     // Show timing per case at end            [UT_SHOW_TIMING]
     bool useInteractive; // Run in interactive mode                [UT_INTERACTIVE]
+    int  timeout;        // Set timeout for child in microseconds  [UT_TIMEOUT]
```

Comment:
Put the units right into the variable name.
As is, a variable called "timeout" can mean multiple things, e.g. It could mean that a timeout occurred, not necessarily that it's the amount of time before something times out.